### PR TITLE
Whats New updates for UG

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) updates What's New from after the 2.4.1 GA release to now. P-rex was already added from #928.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -21,9 +27,69 @@ entries:
   type: Major update
   date: December 7, 2020
   link: https://github.com/magento/merchdocs/pull/914
+- description: Added the ability to [monitor and manage](https://docs.magento.com/user-guide/system/catalog-sync.html)
```
Comment (erikmarr) [inline src/_data/whats-new.yml:30]: Looks like a duplicate

**Review by erikmarr — COMMENTED** [review]
Looks like a duplicate got in there.

File: src/_data/whats-new.yml (hunk 1 above)
Comment (leslietilling) [reply src/_data/whats-new.yml]: Thanks for catching -- fixed. The tool did not like that we put those P-rex items in ahead of time. Thought I caught all of those.

**Review by erikmarr — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**